tapir/shifts/templates/shifts/shiftwatch_overview.html

```diff
@@ -0,0 +1,95 @@
+{% extends "shifts/base.html" %}
+{% load django_bootstrap5 %}
+{% load i18n %}
+{% load core %}
+{% load utils %}
+{% block title %}
+    {% translate "Recurring Shiftwatch List" %}
+{% endblock title %}
+{% block head %}
+    {{ form.media }}
+{% endblock head %}
+{% block content %}
+    <form method="post">
+        {% csrf_token %}
+        <!-- Recurring Shift Watches Card -->
+        <div class="card mb-2">
+            <div class="card-header d-flex justify-content-between align-items-center">
+                <h5>{% translate "Recurring Shift Watches" %}</h5>
+                <div>
+                    <a class="{% tapir_button_link_to_action %}"
+                       href="{% url 'shifts:create_watch_recurring_shifts' user.pk %}"
+                       id="create_recurring_shiftwatch_button">
+                        <span class="material-icons">repeat</span>
+                        {% translate "Create a Rule for recurring Shift-Watches" %}
+                    </a>
+                </div>
+            </div>
+            <div class="card-body">
+                <ul class="list-group list-group-flush">
+                    {% for recurringshiftwatch in recurringshiftwatches %}
+                        <li class="list-group-item">
+                            <input type="checkbox"
+                                   name="recurringshiftwatch_ids"
+                                   value="{{ recurringshiftwatch.id }}">
+                            {% if recurringshiftwatch.shift_template_group and recurringshiftwatch.weekdays %}
+                                {% translate "You are observing all shifts based on the weekdays:" %}
+                                <strong>{{ recurringshiftwatch.weekdays|join:", " }}</strong>
```

Comment:
This shows the weekdays as numbers, showing the days would be nicer. I think WEEKDAY_CHOICES is the usual source of day names.